discojs/discojs-core/src/models/gpt/config.ts

```diff
@@ -50,7 +49,16 @@ export const DEFAULT_CONFIG: Required<GPTConfig> = {
   residDrop: 0.2,
   embdDrop: 0.2,
   tokEmb: true,
-  lmHead: true
+  lmHead: true,
+  nLayer: 3,
+  nHead: 3,
+  nEmbd: 48,
+}
+
+export type ModelSize = {
```

Comment:
it's equivalent but nicer to use `interface` when creating a `type` for an object.

```suggestion
export interface ModelSize {
```

Reply:
I was actually searching for pros and cons and didn't find anything significant, why do you prefer interface?

Reply:
hum, it's a habit mostly, I got it from [one of eslint rule](https://typescript-eslint.io/rules/consistent-type-definitions). now that I reread the documentation, there [aren't much differences](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces). the only plus for interface that I found myself agreeing with it that [it shows better error messages]( https://www.typescriptlang.org/play#example/types-vs-interfaces)